campaignion_csv.module

```diff
@@ -114,3 +114,46 @@ function campaignion_csv_campaignion_csv_info() {
   ];
   return $export;
 }
+
+/**
+ * Implements hook_webform_results_download_submission_information_info_alter().
+ *
+ * Add a NID column right before the SID column.
+ */
+function campaignion_csv_webform_results_download_submission_information_info_alter(array &$info, $node = NULL, $options = NULL) {
+  if (!$options || !empty($options['multiple_nodes'])) {
+    $index = array_search('webform_sid', array_keys($info));
+    $columns['webform_nid'] = t('NID');
+    $info = array_slice($info, 0, $index) + $columns + array_slice($info, $index);
```

Comment:
Isn’t this the same as `array_splice($info, $index, 0, $columns)`?

Reply:
No, this would ignore the array key. (I had that before.)

Reply:
That would remove the array key and use only the value instead.

Reply:
Even if you add the array [key => value] as the value? That's very confusing, but ok…

Reply:
```
php > $a = ['associative' => 'array'];
php > $b = [];
php > array_splice($b, 0, 0, $a);
php > var_dump($b);
array(1) {
  [0]=>
  string(5) "array"
}
```

Reply:
php is great!